sdk/js-query/src/query/response.ts

```diff
@@ -0,0 +1,36 @@
+import { ChainQueryType } from "./request";
+
+// TODO: implement query response parsing
+
+export class QueryResponse {
```

Comment:
This class doesn't do anything.  Do we want to remove it from the index.ts file until it does?

Reply:
Removed